docs/index.md

```diff
@@ -3,174 +3,148 @@
+                    "iam:TagRole",
+                    "iam:ListRolePolicies",
+                    "iam:ListAttachedRolePolicies",
+                    "iam:ListInstanceProfilesForRole",
+                    "iam:AttachRolePolicy",
+                    "iam:DetachRolePolicy"
+                ],
+                "Resource": [
+                    "arn:aws:iam::<ACCOUNT_ID>:oidc-provider/*",
+                    "arn:aws:iam::<ACCOUNT_ID>:role/*"
+                ]
+            }
+        ]
     }
-  }
-}
-
-locals {
-  sts_roles = {
-      role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/ManagedOpenShift-Installer-Role",
-      support_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/ManagedOpenShift-Support-Role",
-      instance_iam_roles = {
-        master_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/ManagedOpenShift-ControlPlane-Role",
-        worker_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/ManagedOpenShift-Worker-Role"
-      },
-      operator_role_prefix = var.operator_role_prefix,
-  }
-}
-
-data "aws_caller_identity" "current" {}
-
-resource "ocm_cluster_rosa_classic" "rosa_sts_cluster" {
-  name           = var.cluster_name
-  cloud_region   = var.region
-  aws_account_id     = data.aws_caller_identity.current.account_id
-  availability_zones = [var.zone]
-  disable_waiting_in_destroy = false
-  properties = {
-    rosa_creator_arn = data.aws_caller_identity.current.arn
-  }
-  sts = local.sts_roles
-}
-
-resource "ocm_cluster_wait" "rosa_cluster" {
-  cluster = ocm_cluster_rosa_classic.rosa_sts_cluster.id
-}
-
-data "ocm_rosa_operator_roles" "operator_roles" {
-  operator_role_prefix = var.operator_role_prefix
-  account_role_prefix = var.account_role_prefix
-}
-
-module operator_roles {
-  source = "terraform-redhat/rosa-sts/aws"
-  version = "0.0.4"
-
-  create_operator_roles = true
-  create_oidc_provider = true
-  create_account_roles = false
-
-  cluster_id = ocm_cluster_rosa_classic.rosa_sts_cluster.id
-  rh_oidc_provider_thumbprint = ocm_cluster_rosa_classic.rosa_sts_cluster.sts.thumbprint
-  rh_oidc_provider_url = ocm_cluster_rosa_classic.rosa_sts_cluster.sts.oidc_endpoint_url
-  operator_roles_properties = data.ocm_rosa_operator_roles.operator_roles.operator_iam_roles
-}
-
-```
\ No newline at end of file
+    ```
+
+## Terraform examples
+
+The following examples show complete, usable Terraform examples. You must configure the examples with your own credentials. After the files are configured, run the following commands to test these examples:
+
+1. `terraform init` - This command uses all the necessary providers for your project.
+1. `terraform plan -out <name>.tfplan` - **Optional**: This command creates the Terraform plan file that you use to create your infrastructure. This command serves as a "dry run" to ensure that the Terraform will execute properly.
```

Comment:
I think the <name> part is supposed to be in italics (https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values).